Rc 3.2.2

dhakman opened the pull request.

dhakman committed:
Add message of cancelled payment

dhakman committed:
Show selected bank on confirm checkout page

dhakman committed:
Merge pull request #24 from paynl/feature/card_0U79to0j_if-a-bank-was-selected-show-in-checkout-form-right-to-payment-method

Show selected bank on confirm checkout page

dhakman committed:
Resolve conflict f9420eb7ea4887a4d78cc3b0fe343d868281523f

dhakman committed:
Merge pull request #23 from paynl/feature/card_fDhbY9VI_show-message-in-case-of-transaction-is-cancelled-or-denied

Add message of cancelled payment

dhakman committed:
Added select which allow user to show payment method's banks

dhakman committed:
Add new images and align description

dhakman committed:
Merge branch 'feature/card_Bv5UHaLS_change-payment-method-images-under-v16-of-sdk' of github.com:paynl/shopware5-plugin into rc-3.2.2

dhakman committed:
Add functionality to show or hide banks

dhakman committed:
Resolve conflict with rc-3.2.2

dhakman committed:
Add Telefonisch betalen image

woutse commented:
dont forget to change field object to 3.2.2

dhakman committed:
Merge pull request #27 from paynl/feature/card_5buqRzgc_add-the-bank-drop-down-make-a-setting-to-activate-this-option-toggle

Added select which allow user to show payment method's banks

woutse reviewed (changes requested):
Dont forget to change object to 3.2.2

dhakman committed:
Added css and ideal issuers for account payment methods

dhakman committed:
Resolve conflict with rc-3.2.2

dhakman committed:
Add Telefonisch betalen icon

dhakman committed:
Load issuer from db

dhakman committed:
Remove spaces for bank

dhakman committed:
Merge pull request #28 from paynl/feature/card_Bv5UHaLS_change-payment-method-images-under-v16-of-sdk

Change payment method images and align description

dhakman committed:
Add dob and phone fields for Billink / AfterPay / Klarna / IN3 payment methods

dhakman committed:
Merge branch 'rc-3.2.2' of github.com:paynl/shopware5-plugin into feature/card_Zr9iA1CA_load-banks-for-ideal-to-account-payment

dhakman committed:
Merge branch 'rc-3.2.2' of github.com:paynl/shopware5-plugin into rc-3.2.2

dhakman committed:
Merge pull request #29 from paynl/feature/card_Zr9iA1CA_load-banks-for-ideal-to-account-payment

Load banks for ideal to account payment

dhakman committed:
Merge branch 'rc-3.2.2' of github.com:paynl/shopware5-plugin into rc-3.2.2

dhakman committed:
Merge rc-3.2.2 release latest commits

dhakman committed:
Ovveride shipping-payment plugin to activate datepicker

dhakman committed:
Added methods to store customer phone number and date of birth

dhakman committed:
Fix code style

dhakman committed:
Send phone and date of birth fields as array with payment methods keys to get the rigth field from request

dhakman committed:
Use plugin subscriber instead of overriding plugin

dhakman committed:
Remove unneeded block

dhakman committed:
Refactored extra fields store proccess. Use separate method storeExtraFields.
Created storeDobAndPhone method to store customer date of birth and phone number

dhakman committed:
Put payment method description to the bottom right part of footer

dhakman committed:
Set fixed width for dob and phone fields on account page

dhakman committed:
Add functionality to adding customer dob and phone at account payment method settings

dhakman committed:
Check if user authorized

dhakman committed:
Fix responsive for payment methods

dhakman committed:
Align fields by pm name

dhakman committed:
Set min max date for dob

dhakman committed:
Code alignment

dhakman committed:
Merge pull request #31 from paynl/feature/card_hzWUijMD_put-payment-method-description-in-the-rigth-bottom-block-when-we-change-it

Put payment method description in the rigth bottom block when we change it

dhakman committed:
Remove unneded getUserAttributeData method and javascript prop

dhakman committed:
Resolve conflicts with rc-3.2.2

dhakman committed:
Merge pull request #30 from paynl/feature/card_8rcsf9Hc_add-phone-number-and-date-of-birth-to-checkout-form-in-case-one-of-these-billink-afterpay-klarna-in3-payment-methods-was-selecte

Added phone number and date of birth fields to checkout form in case of one of these payment methods: billink, afterpay, klarna, in3

dhakman committed:
Resolve conflicts with rc-3.2.2

dhakman committed:
Add check on account page whether allowed showing bank list

dhakman committed:
Show br tag in case if we have to show dob

dhakman committed:
Break functionality on separate methods

dhakman committed:
Use cons for getting plugin name

dhakman committed:
Rename method

dhakman committed:
Add comments

dhakman committed:
Added dob and phone fields for spraypay / creditclick

dhakman committed:
Merge pull request #33 from paynl/feature/card_sTPDUcnX_add-dob-and-phone-fields-for-spraypay-creditclick

Add dob and phone fields for spraypay creditclick

dhakman committed:
Set min 7 version of php and version plugin 3.2.2

dhakman committed:
Set release version 3.3.0

dhakman committed:
Merge pull request #34 from paynl/feature/card_d6OFMGix_update-composerjson-to-use-560-as-min-version-we-would-like-to-drop-support-5-version-of-php

Set min support php version 7 and new release version 3.3.0

dhakman committed:
Merge pull request #32 from paynl/feature/card_MRnrhK2B_refactor-rc-322

Refactor rc 322

dhakman committed:
Set 5.6 version of shopware as min

dhakman committed:
Merge pull request #35 from paynl/feature/card_d6OFMGix_update-composerjson-to-use-560-as-min-version-we-would-like-to-drop-support-5-version-of-php

Set min compatible version 5.6.0 of shopware
